### PR TITLE
Update ColorManager.php

### PR DESCRIPTION
**Pull request by schrempfra**
change rgb to rgba

### CODE REVIEW

File: src/Facades/ColorManager.php [hunk 1]
```diff
@@ -31,7 +31,7 @@ public static function getColorAsArray(array $options, string $optionKey): array
             try {
                 $rgbColor = Rgb::fromString($getColor);
             } catch (\Exception) {
-                $rgbColor = Rgb::fromString(Qr::getDefaultOptions()[$optionKey]);
+                $rgbColor = Rgba::fromString(Qr::getDefaultOptions()[$optionKey]);
```
Comment (atmonshi) [inline src/Facades/ColorManager.php]: why using `Rgba` for none Rgba colors? notice the 

`if (str($getColor)->startsWith('rgba')) {`

some colors dont use rgpa

Comment (schrempfra) [reply src/Facades/ColorManager.php]: Yes, we need to implement a color format check

Comment (atmonshi) [reply src/Facades/ColorManager.php]: ya but the condition is way up there line 22 :) it will never reach this part! and it already has its own try catch

Comment (schrempfra) [reply src/Facades/ColorManager.php]: Sure it reaches this part of code, if you type in some random strings

Comment (schrempfra) [reply src/Facades/ColorManager.php]: My new commit solves the problem with the distinction between rgb and rgba colors

**Comment by atmonshi** [comment]
thank you

**Pull request merged**